src/Voice.tsx

```diff
@@ -600,16 +600,23 @@ export class Voice extends EventEmitter {
    * @param token - A Twilio Access Token, usually minted by an
    * authentication-gated endpoint using a Twilio helper library.
    * @param params - Custom parameters to send to the TwiML Application.
+   * @param contactHandle - (iOS only) a CallKit display name that will show in
+   * the call history as the contact handle. Optional.
```

Comment:
What happens if this is not passed on iOS? What happens if this is passed on Android? Should customers always pass this if they can, or should they use platform specific code?